### DISCUSSION TITLE
DIPY Project 5: using AI/ML in Diffusion MRI processing

### DISCUSSION THREAD

**Discussion by NeuralFlux** [dipy/dipy#3067] [#GSoC/GSoD]
Hi! I'm Anudeep, a Computer Science graduate at New York University, and I am keen to explore ideas around project 5. I have a solid grasp of ML theory and PyTorch, and I am currently a Graduate Research Assistant working on reinforcement learning algorithms for crypto trading. I have used NNs like Transformers, LSTMs, and CNNs for various applications—food label classification, RL, essay grading.  I particularly find [synthetic MRI generation](https://gist.github.com/lb-97/57347e7d06d87a0aa3b77887631f33bc) from GSoC 2023 interesting. May I please know if you are looking for more ideas or have something you'd like to explore?

@pjsjongsung @sreekarchigurupati @skoudoro

**Comment by pjsjongsung** [comment]
Hello. Thank you for your interest. We already have a team working on the project, so unfortunately synthetic images is not an option. However, if you have any ideas on other diffusion MRI related Deep Learning projects, feel free to let us know!

**Comment by NeuralFlux** [comment]
Thank you for the response. I will share any interesting applications I find!

**Comment by Saproo** [comment]
Thank you for the information, this will help me in many cases.

**Comment by skoudoro** [comment]
Hi @Saproo,

Please, stop to spam every discussion and take the time to read the rules,  code of conduct, how to contribute files, the GSoC wiki.

if you continue we will have to ban you from the repo. 

Concerning the workshop you just need to click the link to get all your answer. 

thank you 

ps: It is normal to ask question and every question are welcomed. However, it is your job to find the relevant information before asking the relevant question.

**Comment by Saproo** [comment]
Sorry.

**Comment by Saproo** [comment]
Respected sir,
Can you help me out with gsoc 2026, I am a first year Btech student doing
Computer Science and Engineering.
Thanks & Regards
Saptarshi Sen
